standby/src/lib.rs

```diff
@@ -922,7 +900,7 @@ impl Standby {
 
     /// Process a general event that is either of a particular type or in a
     /// particular guild.
-    #[cfg_attr(feature = "tracing", tracing::instrument(level = "trace"))]
+    #[tracing::instrument(level = "trace")]
```

Comment:
Only `trace` level events are recorded inside, so there's no need for the instrument to be at the trace level.
```suggestion
    #[tracing::instrument]